Fluid/Filters/ArrayFilters.cs

```diff
@@ -103,6 +103,45 @@ public static async ValueTask<FluidValue> Map(FluidValue input, FilterArguments
             return new ArrayValue(list);
         }
 
+        public static ValueTask<FluidValue> Reverse(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            if (input.Type == FluidValues.Array)
+            {
+                return new ArrayValue(input.Enumerate().Reverse());
+            }
+            else if (input.Type == FluidValues.String)
+            {
+                var value = input.ToStringValue();
+                if (String.IsNullOrEmpty(value))
+                {
+                    return StringValue.Empty;
+                }
+                else
+                {
+                    String reversedString;
+                    var valueAsArray = value.ToCharArray();
+#if NETSTANDARD2_0
+                    Array.Reverse(valueAsArray);
+
+                    reversedString = new String(valueAsArray);
```

Comment:
Wrong. I mentioned that in another comment, this filter when applied on strings actually returns an array of chars, not a string.

Reply:
Oops seems I forgot to push the local changes